README.md

```diff
@@ -26,21 +26,21 @@ If you are new to PowerShell and would like to learn more, we recommend reviewin
 
```

Comment:
@adityapatwardhan, your last commit had 8 failures in `PowerShell-CI-macos`
(These are 5 of the failures)

PackageManagement Acceptance Test.find-packageprovider PowerShellGet
```powershell
Expected 'PowerShellGet' to be found in collection @(), but it was not found.
at <ScriptBlock>, /Users/runner/runners/2.165.0/work/1/s/test/powershell/Modules/PackageManagement/PackageManagement.Tests.ps1: line 41
41:         $fpp | Should -Contain "PowerShellGet"
```
PackageManagement Acceptance Test.install-packageprovider, Expect succeed
```powershell
Expected 'gistprovider' to be found in collection @(), but it was not found.
at <ScriptBlock>, /Users/runner/runners/2.165.0/work/1/s/test/powershell/Modules/PackageManagement/PackageManagement.Tests.ps1: line 46
46:         $ipp | Should -Contain "gistprovider"
```
PackageManagement Acceptance Test.Find-package
```powershell
Expected 'jquery' to be found in collection @(), but it was not found.
at <ScriptBlock>, /Users/runner/runners/2.165.0/work/1/s/test/powershell/Modules/PackageManagement/PackageManagement.Tests.ps1: line 51
51:         $f.Name | Should -Contain "jquery"
```
PackageManagement Acceptance Test.Install-package
```powershell
Expected 'jquery' to be found in collection @(), but it was not found.
at <ScriptBlock>, /Users/runner/runners/2.165.0/work/1/s/test/powershell/Modules/PackageManagement/PackageManagement.Tests.ps1: line 56
56:         $i.Name | Should -Contain "jquery"
```
PackageManagement Acceptance Test.Get-package
```powershell
Expected 'jquery' to be found in collection @(), but it was not found.
at <ScriptBlock>, /Users/runner/runners/2.165.0/work/1/s/test/powershell/Modules/PackageManagement/PackageManagement.Tests.ps1: line 61
61:         $g.Name | Should -Contain "jquery"
```